low-level/framework/base/sspl_constants.py

```diff
@@ -164,6 +164,33 @@
 
 SSPL_CONFIGS = ['log_level', 'cli_type', 'sspl_log_file_path', 'cluster_id', 'storage_enclosure', 'setup', 'operating_system']
 
+MODULES = {"CPUFAULTSENSOR": {"probe" : "sysfs", "threaded": "true"},
+            "IEMSENSOR" : {"log_file_path": "/var/log/cortx/iem/iem_messages", "threaded" : "true", "timestamp_file_path" : "/var/cortx/sspl/data/iem/last_processed_msg_time"},
+            "IPMI": {"pass": "admin", "user": "admin"},
```

Comment:
> "IPMI": {"pass": "admin", "user": "admin"},

Is this used by any code today(check nodehw sensor for ipmi over lan feature). If not, remove from the R2 config.

Same for following
> "LOGEMAILER" : {"priority" : "LOG_ERR"},
"LOGGINGMSGHANDLER" : {"iem_log_locally" : "true", "iem_routing_enabled" : "false"},
"RARITANPDU": { "IP_addr" : "172.16.1.222", "comm_port" : "/dev/ttyACM0", "max_login_attempts" : "2", "pass" : "admin","user" : "admin"},
"SMRDRIVEDATA" : {"logging_interval" : "3600", "threaded": "true"},
"SMTPSETTING" : {"recipient" : "example@seagate.com", "smptserver" : "mailhost.seagate.com", "smtp_port" : "25"},